components/desktop/mate/mate-themes/Makefile

```diff
@@ -16,13 +16,13 @@
 include ../../../../make-rules/shared-macros.mk
 
 COMPONENT_NAME=		mate-themes
-COMPONENT_VERSION=	3.20.7
+COMPONENT_VERSION=	3.20.19
 COMPONENT_PROJECT_URL=	http://www.mate-desktop.org
 COMPONENT_SUMMARY=	MATE Themes
 COMPONENT_SRC=		$(COMPONENT_NAME)-$(COMPONENT_VERSION)
 COMPONENT_ARCHIVE=	$(COMPONENT_SRC).tar.xz
 COMPONENT_ARCHIVE_HASH=	\
-    sha256:c2909366b83e9e56e630fd5f51c5cf2f6993e0eb337717cac2663c708577606e
+    sha256:0fad212926a59aedbeaf45d5e96a2337cebd54cc17ced38f5e50ce46ce786885 
 COMPONENT_ARCHIVE_URL=	http://pub.mate-desktop.org/releases/themes/3.20/$(COMPONENT_ARCHIVE)
```

Comment:
Doesn't publish:
`` Couldn't find 'usr/share/themes/Menta/gtk-3.0/assets/slider-hover.png' in any of the specified search directories:``
and so on. Have you updated manifests and sample-manifests?